root/components/common-macros.tt

```diff
@@ -264,7 +264,7 @@ END -%]
       SET text = html_escape(entity.name) IF text == '';
       SET text = html_escape(l('[removed]')) IF text == '';
       caption = allow_new
-        ? l("This entity will be created when edits are entered.")
```

Comment:
It appears this message does make sense if you're adding a new medium with new recordings: even if you make the add-medium edit votable, the recordings are created immediately when the edit is entered.

But if you're *editing* a medium to use a new recording, it's not created until the edit is applied, so the message below makes more sense.

Reply:
Hmm. "This entity will be created during the editing process" is probably more precise, then, but it still suggests "when entering the edit" to me. Any ideas?

Reply:
"This entity is pending creation with the edit?" Or we can have two separate messages controlled with `allow_new_before_entered` and `allow_new_before_applied` if we want to keep the distinction.

Reply:
Updated the wording to be more general.